### PR TITLE
fix: special-case media-playlist-item's type attr

### PR DESCRIPTION
**Pull request by gkatsev**
The type attribute is a special attribute to tell media-playlist which media-slotted element to use.

Some media-slotted elements might use a `type` attribute for something and so we probably don't want to copy it over.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <!-- <script type="module" src="index.js"></script> -->
     <script type="module" src="https://unpkg.com/hls-video-element@0"></script>
+    <script type="module" src="https://unpkg.com/@mux/mux-video@0/dist/mux-video.mjs"></script>
```
Comment (luwes) [inline index.html]: could this be changed to jsDelivr?

**Review by luwes — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (gkatsev) [reply index.html]: Yes, I just kept it as unpkg as that's what the other URL was.

Comment (gkatsev) [reply index.html]: Updated.

**Pull request merged**